app/controllers/jobs_controller.rb

```diff
@@ -3,6 +3,12 @@ class JobsController < ApplicationController
 
   def index
     @jobs = Job.all
```

Comment:
are you using @jobs anymore or can you remove?

Reply:
oh this can be removed